examples/network/aws/http-aws/manifest.json

```diff
@@ -0,0 +1,27 @@
+{
+	"include": [
+		"$(MODDABLE)/examples/manifest_base.json",
+		"$(MODDABLE)/examples/manifest_net.json",
+		"$(MODDABLE)/modules/crypt/tls.json"
+	],
+	"modules": {
+		"*": [
+			"./main",
+			"$(MODULES)/network/http/*"
+		]
+	},
+	"preload": [
+		"http"
+	],
+	"resources": {
+		"*": [
+            "./CA1-key",
```

Comment:
This line appears to be using spaces for indenting while all other lines use tabs.